examples/illustrations/database_records/database_records_validation.ttl

```diff
@@ -6,6 +7,39 @@
 
 []
 	a sh:ValidationReport ;
-	sh:conforms "true"^^xsd:boolean ;
```

Comment:
The examples need to be updated to reflect the new `drafting.ttl` state.  @kchason , are you able to do this and run `make` to regenerate these?